src/plugins/discover/public/application/angular/directives/fixed_scroll.js

```diff
@@ -1,155 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import $ from 'jquery';
-import _ from 'lodash';
-import { createDebounceProviderTimeout } from './debounce';
-
-const SCROLLER_HEIGHT = 20;
-
-/**
- * This directive adds a fixed horizontal scrollbar to the bottom of the window that proxies its scroll events
- * to the target element's real scrollbar. This is useful when the target element's horizontal scrollbar
- * might be waaaay down the page, like the doc table on Discover.
- */
-export function FixedScrollProvider($timeout) {
```

Comment:
`.dscTableFixedScroll` is also legacy and I've removed it now. We don't need this anymore, since the table is now in a scrollable div, so there is a horizontal scrollbar that's not displayed way down the page. which was the reason for the fixed_scroll functionality.